lib/matplotlib/cbook/__init__.py

```diff
@@ -1914,20 +1914,28 @@ class so far, an alias named ``get_alias`` will be defined; the same will
     if cls is None:  # Return the actual class decorator.
         return functools.partial(_define_aliases, alias_d)
 
-    def make_alias(name):  # Enforce a closure over *name*.
-        def method(self, *args, **kwargs):
-            return getattr(self, name)(*args, **kwargs)
-        return method
+    def make_alias(method):
+        import inspect
+        args = str(inspect.signature(method))[1:-1]  # remove parentheses
```

Comment:
`str(inspect.signature(functools.partial(method, None)))` will bind the first argument for you and remove it from the signature more easily than the text wrangling below :) (plus it even works if someone decides to not call the first argument "self", hehe... for example it could just be a wrapper function itself being `*args, **kwargs`)